modules/misc/news.nix

```diff
@@ -2252,6 +2252,16 @@ in
           '~/.taskrc'.
         '';
       }
+
+      {
```

Comment:
I think this change is worth mentioning in the release notes as well!

Reply:
I added an entry in the 22.05 release note.